content/en/blog/2023/end-user-discussions-02.md

```diff
@@ -225,5 +225,5 @@ organization, we’d love to hear from you! Ways to share:
 
 Be sure to follow OpenTelemetry on
 [Mastodon](https://fosstodon.org/@opentelemetry) and
-[Twitter](https://twitter.com/opentelemetry), and share your stories using the
+[X](https://x.com/opentelemetry), and share your stories using the
```

Comment:
```suggestion
[X](https://x.com/opentelemetry), previously known as Twitter, and share your stories using the
```

Same for the rest. Most mainstream media outlets are doing it this way and I concur.